garrysmod/gamemodes/base/entities/entities/npc_tf2_ghost.lua

```diff
@@ -66,7 +66,7 @@ end
 -- List the NPC as spawnable
 --
 list.Set( "NPC", "npc_tf2_ghost", {
-	Name = "Example NPC",
+	Name = "#npc_tf2_ghost",
 	Class = "npc_tf2_ghost",
-	Category = "Nextbot"
+	Category = "#spawnmenu.npcs.category.nextbot"

```

Comment:
I really dislike this NPC, and these localization strings.

The name string could probably be set to "#npc_example` and the category probably shouldn't be translated, since `Nextbot` is just name of the internal technology, not a real word. Perhaps this NPC should be put into `Other` category even.

Reply:
Hmm I still want it to be "transliterated". Other languages may adopt the word using their respective alphabets.

Reply:
> Hmm I still want it to be "transliterated". Other languages may adopt the word using their respective alphabets.

While I believe that it can be transliterated in some languages (_not in mine, as I found out_ 😅), there are two things:
1. Looks like It should be `NextBot` instead of `Nextbot`.
2. I really like the idea of moving it to the "Other" category.